VocaDb.Migrations/Migrations.cs

```diff
@@ -1,10 +1,149 @@
 using System.Data;
+using System.Linq;
 using FluentMigrator;
 
 namespace VocaDb.Migrations
 {
 	// Migration version format: YYYY_MM_DD_HHmm
 
+	[Migration(2020_11_20_2000)]
+	public class Comments : Migration
+	{
+		private sealed class CommentTable
+		{
+			public CommentTable(string name, string schema = null, string entryTypeName = null)
+			{
+				Name = name;
+				Schema = schema;
+				EntryTypeName = entryTypeName;
+			}
+
+			/// <summary>
+			/// Table name without schema. (e.g. `AlbumComments`, `ArtistComments`, `DiscussionComments`, ...)
+			/// </summary>
+			public string Name { get; }
+
+			/// <summary>
+			/// Table schema. (either null or `discussions`)
+			/// </summary>
+			public string Schema { get; }
+
+			/// <summary>
+			/// Entry type name. (e.g. `Album`, `Artist`, `Topic`, ...)
+			/// </summary>
+			public string EntryTypeName { get; }
+
+			/// <summary>
+			/// Table name with schema. (e.g. `AlbumComments`, `ArtistComments`, `discussions.DiscussionComments`, ...)
+			/// </summary>
+			public string NameWithSchema => Schema != null ? $"{Schema}.{Name}" : Name;
+		}
+
+		public override void Up()
+		{
+			// Rename columns on `DiscussionTopics` table.
+			Rename.Column("Content").OnTable(TableNames.DiscussionTopics).InSchema(SchemaNames.Discussions).To("Message");
+
+			// Rename columns on `AlbumReviews` table.
+			Rename.Column("Date").OnTable(TableNames.AlbumReviews).To("Created");
+			Rename.Column("Text").OnTable(TableNames.AlbumReviews).To("Message");
+			Rename.Column("User").OnTable(TableNames.AlbumReviews).To("Author");
+			Delete.Index("UX_AlbumReviews").OnTable(TableNames.AlbumReviews);

```

Comment:
Deleted `UX_AlbumReviews` to implement soft deletes for reviews.

Reply:
Perhaps you could add it as normal non-unique index instead?